### ISSUE TITLE
redis-cluster 0.0.1 gone

### ISSUE THREAD

**Issue by dano0b** [inspur-iop/charts#1]
First, thanks a lot for the redis-cluster helm chart! Sadly version 0.0.1 is gone which breaks our pipeline for our application. We added
```
  - name: redis-cluster
    version: 0.0.1
    repository: https://inspur-iop.github.io/charts/
    condition: redis-cluster.enabled
```
to our helm chart and it looks that this version is gone.

**Comment by wilsonlindelof** [comment]
https://github.com/inspur-iop/charts/commit/29f8c6c98321f04875b923e81b68443e443e9dd0#diff-7ef2a3e809cfbd83be97d22711284407 its been updated to 0.0.2. The only change is to the readme: https://github.com/inspur-iop/charts-src/commits/master/incubator/redis-cluster

**Comment by dano0b** [comment]
Yeah, actually happy that it is maintained. Just surprised that the previous version is not available anymore in the chart repository. I guess it happened since the git repository changed. I hope the next versions will be additional not replacing existing ones.
